src/.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+
+node_js:
+  - node
+
+services:
+  - postgresql
+  
+before_script:
+  - psql -c 'create database DB_URL;' -U postgres
```

Comment:
This won't work because `DB_URL` is an absolute url containing a protocol, username, password and database name.

Reply:
Should I put inside before_script? just before creating a database?

Reply:
Where is the `migrate` script defined? I can't find it in `develop` or this branch.

The migrate command should be after creating the database